### PR TITLE
Show Chromium, Electron & Node version in about window

### PR DESCRIPTION
**Pull request by linonetwo**
Let's discuss in Telegram before merge!

Screen Shot 2020-04-18 at 7 51 04 PM

### CODE REVIEW

File: template/src/components/dialog-about/index.js [hunk 1]
```diff
@@ -67,6 +68,16 @@ const About = (props) => {
         >
           {`Version v${appVersion}. Powered by WebCatalog.`}
         </Typography>
+        <Typography variant="body2" className={classes.version}>
+          {versions.map(({ name, version }) => (
+            <div>
```
Comment (quanglam2807) [inline template/src/components/dialog-about/index.js]: for list in React, you should assign a `key` to every item. In this case, it's not that useful but in many other cases, it'll help avoiding many bugs.

File: template/public/preload/about.js [hunk 2]
```diff
@@ -10,3 +10,9 @@ const contextMenu = require('electron-context-menu');
 contextMenu({
   window: remote.getCurrentWindow(),
 });
+
+window.getVersions = () => [
```
Comment (quanglam2807) [inline template/public/preload/about.js]: You can using `process` directly in React code by adding `window.`. For example: `window.process.versions.electron` so this code can be moved to `src`

File: template/src/components/dialog-about/index.js [hunk 3]
```diff
@@ -67,6 +68,16 @@ const About = (props) => {
...
+              {name}
+              :
+              {' '}
+              {version}
+            </div>
```
Comment (quanglam2807) [inline template/src/components/dialog-about/index.js]: `<Typography>` is `<p>`, so it doesn't like that you have `<div>` as its children.

File: template/src/components/dialog-about/index.js (hunk 1 above)
Comment (linonetwo) [reply template/src/components/dialog-about/index.js]: Oh, I thought eslint will warn me...

**Pull request merged**